sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java

```diff
@@ -11,7 +11,10 @@
 
   @SuppressWarnings("AnnotateFormatMethod")
   @Override
-  public void log(SentryLevel level, String message, Object... args) {
+  public void log(
+      final @Nullable SentryLevel level,
```

Comment:
Perhaps these parameters should not be `nullable`?

Reply:
Agreed, can't think of a use case were we can't provide a log here and we really need one to make sense of things.
Otherwise we'll need to add some default and odd stuff happens